src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -1743,6 +1953,15 @@
                 "acl_counter_high_threshold": "85",
                 "acl_counter_low_threshold": "70",
                 "acl_counter_threshold_type": "percentage",
+                "snat_entry_threshold_type": "percentage",
```

Comment:
kindly remove it from here because this config is just to test an unknown table.